### ISSUE TITLE
docker run -v flag dose not working when using docker client

### ISSUE THREAD

**Issue by Qinmu-Xie** [rancher/os#1018]
I'm running  RancherOS 0.4.5 on docker-machine.
I use docker client  (eval $(docker-machine env $MACHINE-NAME)) to run docker cmd.
and found that **docker run -v flag is not working**.
but it **works fine when I log into rancherOS** (docker-machine ssh $MACHINE-NAME)
I don't know why this happening and have no idea where to find answers.

**Comment by joshwget** [comment]
In what way is it not working? Can you mention how you'd expect this to work?

**Comment by Qinmu-Xie** [comment]
Sorry I have not made it clear.
When I use **docker run -it -v $PWD:/home/data ubuntu /bin/bash** 
I expect that there should be files in container's /home/data directory.
That works fine after I use docker-machine ssh XXX in the host rancherOS(XXX).
But When I use docker client to run that commend, there is just a /home/data directory in container, but no files in it.

**Comment by SvenDowideit** [comment]
ah, this is due to docker-machine and rancherOS not setting up the host-mounts (I'm assuming @Qinmu-Xie is using virtualbox? )

**Comment by joshwget** [comment]
@SvenDowideit Yeah, I think this is a duplicate of #143 to an extent. Seems like file sharing is what @Qinmu-Xie is expecting to work.
